src/server/app_render.jsx

```diff
@@ -12,7 +12,8 @@ import {determineViewMode} from "../app/utils/Links";
 
 const path = require('path');
 const ROOT = path.join(__dirname, '../..');
-const DB_RECONNECT_TIMEOUT = process.env.NODE_ENV === 'development' ? 1000 * 60 * 60 : 1000 * 60 * 10;
+
+const DB_RECONNECT_TIMEOUT = process.env.NODE_ENV !== 'production' ? 1000 * 60 * 60 : 1000 * 60 * 10;
```

Comment:
shouldn't `DB_RECONNECT_TIMEOUT` just be defined directly in the configuration without the conditional expression?

Reply:
yes. I'll do that as it isn't something that occurred to me.

Reply:
@gl2748 ping